### ISSUE TITLE
是否可以将MAA及其依赖项打包成.msi程序？

### ISSUE THREAD

**Issue by zyfan42** [MaaAssistantArknights/MaaAssistantArknights#9031]
### 在提问之前...

- [X] 我填写了简短且清晰明确的标题，以便开发者在翻阅 issue 列表时能快速确定大致问题。而不是“一个建议”、“卡住了”等
- [X] 我基本确定这是一个新功能/建议，而不是遇到了 bug（不确定的话请附上日志）

### 说说你遇到的问题？

解压并直接在对应的目录中运行MAA**似乎不是**一种标准化的方式，也会给部分用户在**应用部署和依赖项管理**等方面造成困难。

### 有什么好的想法？

1. **标准化安装过程**: 使用 .msi 程序可以提供标准化的安装和卸载过程，用户可以通过通用的安装向导来完成安装，无需手动操作文件或注册表。
2. **版本控制**: .msi 文件可以包含版本信息，使得在安装时可以检查当前系统中是否已经安装了相同版本的应用程序，从而确保不会重复安装或降级。
3. **依赖项管理**: .msi 文件可以包含应用程序的所有依赖项，确保安装时一次性安装所有必要的组件，减少了用户手动配置的复杂性。
4. **可靠性**: Windows Installer 提供了一系列的安装、卸载和维护功能，例如事务支持、回滚机制等，确保安装过程的可靠性和一致性。

**Comment by Constrat** [comment]
1. MAA installation is as simple as it can already be. You get a .zip file. Just extract it and you're done. To uninstall just delete the folder. The config files are all located in the same directory (AppData is not touched). To make MAA work there has never been a need to modify files and / or touch the Registry. 
2. MAA is made with the possibility to be run along various instances of itself. Download more MAA to have more MAA running. Having an installer would just complicate this.
3. This is the only "issue" at the moment. MAA is reliant on .NET libraries, but so are most of programs (and games) and sometimes you will have to manually download .NET libraries
4. The current "installation" process is as simple and straightforward as it can be. It's literally download and unzip. The end.

This is my completely honest and personal opinion, but if I had to choose between an installer (.msi) or a portable (.zip) installation method I would always go with the portable, as it's the simplest to complete remove, with the added benefit that you can just copy paste your entire /MAA directory wherever you want and you will have a working MAA instance.

TL;DR: personally portable zip > installer msi

**Comment by zyfan42** [comment]
You make a good point. That was actually something I thought of when I was helping my newbie friend set it up.  ( I even tried it myself.
